controllers/binding/snapshotenvironmentbinding_adapter_test.go

```diff
@@ -501,14 +503,30 @@ var _ = Describe("Binding Adapter", Ordered, func() {
 				ContextKey: loader.ApplicationContextKey,
 				Resource:   hasApp,
 			},
+			{
+				ContextKey: loader.ComponentContextKey,
+				Resource:   hasComp,
+			},
 			{
 				ContextKey: loader.EnvironmentContextKey,
 				Resource:   hasEnv,
 			},
+			{
+				ContextKey: loader.SnapshotContextKey,
+				Resource:   hasSnapshot,
+			},
 			{
 				ContextKey: loader.DeploymentTargetContextKey,
 				Resource:   deploymentTarget,
 			},
+			{
+				ContextKey: loader.DeploymentTargetClaimContextKey,
+				Resource:   deploymentTargetClaim,
+			},
+			{
+				ContextKey: loader.SnapshotEnvironmentBindingContextKey,
+				Resource:   hasBinding,
+			},
 		})
 
 		dtc, _ := adapter.loader.GetDeploymentTargetClaimForEnvironment(k8sClient, adapter.context, hasEnv)
```

Comment:
Since the `var dtc` definition was introduced above, the source check is complaining about this assignment:
```
no new variables on left side of :=
```
Changing to `=` should fix it.

Reply:
Ah, already taken care of, thanks @jinqi7 !